packages/upstream/nocrypto.0.5.3/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "1.2"
```

Comment:
If we wanted to use `nocrypto` `0.5.4`, we'd have to upgrade to `cstruct` `2.4.0`. I think this is fine, I'll upgrade this lib anyway in my xs-opam update, which contains the newest cstruct.

Reply:
We have bad experiences with upgrades and downgrades, I'd rather stay with this version and bundle the updates in the big PR you are working on for after the next xenserver release

Reply:
Yes, I agree: the 0.5.4 nocrypto doesn't seem to have anything we need, and upgrading cstruct would meaning having to recompile lots of other things. (That's why I chose nocrypto 0.5.3 for this pull-request.)

Reply:
I've upgraded it in my new `xs-opam` anyway :smile: - will push it into a `next` branch of the main `xs-opam`.